assets/xml/scenes/overworld/spot16.xml

```diff
@@ -2,6 +2,11 @@
     <File Name="spot16_scene" Segment="2">
         <Cutscene Name="gDMTOwlCs" Offset="0x1E6A0"/>
         <Cutscene Name="gDMTIntroCs" Offset="0x7EA0"/>
+        <Cutscene Name="gDMTKokiriEmeraldPart6Cs" Offset="0x3D40"/>
+        <Cutscene Name="gDMTGoronRubyCs" Offset="0x40B0"/>
+        <Cutscene Name="gDMTUnusedCs" Offset="0x63D0"/>
+        <Cutscene Name="gDMTCreditsSixSagesCs" Offset="0x66E0"/>
+        <Cutscene Name="gDMTCreditsCameraPanCs" Offset="0x6550"/>
         <Path Name="spot16_scenePathList_000254" Offset="0x254" NumPaths="2"/>
```

Comment:
any more descriptive names for this one? I have no idea what this is by this name

Reply:
oh I missed the word "credits", that helps at least

Reply:
yeah this one is the camera pan towards goron city during the credits, I didn't want to name it `gDMTCreditsCs` because of the part with the sages

Reply:
I would probably prefer `gDMTCreditsShowGoronCityCs` or something along those lines, so that you at least have an idea of what scene it is from the name

Also should we still be naming these as part numbers? Im not sure what direction we are going with for that

Reply:
I thought the same but I think all of the credits cutscenes doesn't have numbers and parts, imo it's fine if it's not 100% consistent with everything else, the point of these PRs is giving descriptive names to the cutscenes that needs it, I'm not really focusing on the consistency